dungeon-crawler/android/src/main/java/com/novoda/dungeoncrawler/GamePauseObservable.java

```diff
@@ -0,0 +1,56 @@
+package com.novoda.dungeoncrawler;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+class GamePauseObservable {
```

Comment:
should this have firebase or remote in the name?

Reply:
Done in 0f611b7100f1d60cf249885232b58d7c4e43d5f8